templates/map.html

```diff
@@ -365,82 +365,6 @@ <h3>Weather Alerts</h3>
             </div>
           </div>
 
-          <h3><i class="fa fa-location-arrow fa-fw"></i>Location &amp; Search Settings</h3>
-          <div>
-              {% if show.fixed_display %}
-            <div class="form-control switch-container">
-              <label for="next-location">
-              <h3>Change search location</h3>
-              <input id="next-location" type="text" name="next-location" placeholder="Change search location">
-              </label>
-            </div>
-            <div class="form-control switch-container">
-              <h3>Lock search marker</h3>
-              <div class="onoffswitch">
-                <input id="lock-marker-switch" type="checkbox" name="lock-marker-switch" class="onoffswitch-checkbox" checked/>
-                <label class="onoffswitch-label" for="lock-marker-switch">
-                <span class="switch-label" data-on="On" data-off="Off"></span>
-                <span class="switch-handle"></span>
-                </label>
-              </div>
-            </div>
-            <div class="form-control switch-container">
-              <h3>Search follows me</h3>
-              <div class="onoffswitch">
-                <input id="geoloc-switch" type="checkbox" name="geoloc-switch" class="onoffswitch-checkbox" checked/>
-                <label class="onoffswitch-label" for="geoloc-switch">
-                <span class="switch-label" data-on="On" data-off="Off"></span>
-                <span class="switch-handle"></span>
-                </label>
-              </div>
-            </div>
-              {% endif %}
-            <div class="form-control switch-container" >
-              <h3>Start map at my position</h3>
```

Comment:
`Start map at my position` and `Follow me` might be used today. Wouldn't it be better to keep them?

Reply:
I have to admit. I've tested it and it didn't work for me. Even on https with location API being enabled. I'll have another look into it.

Reply:
I don't know what are the conditions to make it work, but on my instance it works. Do you want me to send you the URL?